cleancode/src/main/java/com/epam/cleancode/comments/MortgageInstallmentCalculator.java

```diff
@@ -1,39 +1,52 @@
 package com.epam.cleancode.comments;
 
 public class MortgageInstallmentCalculator {
+    private final static String NEGATIVE_VALUES_ARE_NOT_ALLOWED = "Negative values are not allowed";
+    private static double rateOfInterestInDecimal;
```

Comment:
Do we really need this variables? (rateOfInterestInDecimal, termOfMortgageInMonths monthlyRate)

Reply:
we were asked to make the code as clear as possible. i did it.
tried to make it scalable. for ease of viewing .
i made variable not local for ease read code, but it can  be local.(or may without it)

Reply:
Parameter name and name inside function is enough , no need to create static variables

Reply:
ok

Reply:
> 
> 
> Parameter name and name inside function is enough , no need to create static variables

must remake this moment ?
or it is simply desirable to take this into in the future

Reply:
done